### PR TITLE
Type hinting Added to function and methods

### PR DESCRIPTION
**Pull request by mshemuni**
Type hinting added. see: https://docs.python.org/3/library/typing.html

### CODE REVIEW

File: html_table_parser/parser.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def __init__(
         self.named_tables = {}
         self.name = ""
 
-    def handle_starttag(self, tag, attrs):
+    def handle_starttag(self, tag: str, attrs: str) -> None:
```
Comment (davidov541) [inline html_table_parser/parser.py]: attrs should be a list, not a str here.

**Review by davidov541 — CHANGES_REQUESTED** [review]
Just one place that looks wrong. Otherwise thanks for taking care of this to make it a bit easier to use!

File: html_table_parser/parser.py (hunk 1 above)
Comment (mshemuni) [reply html_table_parser/parser.py]: Thank you for the request. I fixed it.

One can ask why I didn't use `list` and imported the `List` from `typing`. The reason is that "deep type hinting" cannot be done with the built-in `list` in `python3.5`.

**Review by davidov541 — APPROVED** [review]

**Comment by davidov541** [comment]
Thank you for your submission! I'm going to merge this now. 

@schmijos - I'm not sure what it takes to get a new version of this available in pip. Can you take care of that or link me to a guide on how to do it for myself?

**Comment by schmijos** [comment]
@davidov541 Do you have a Pypi user, I could add for managing releases?

**Comment by mshemuni** [comment]
> @davidov541 Do you have a Pypi user, I could add for managing releases?

Looking at the pypi page it looks like someone else uploaded it.
We should find that person.

**Comment by schmijos** [comment]
I've got owner access to that package on Pypi. So that should be fine.

**Comment by davidov541** [comment]
@schmijos - My pypi account is @davidov541.

**Comment by schmijos** [comment]
@davidov541 your pypi ownership invite is pending now. Cheers and thank you!

**Pull request merged**